### PR TITLE
Upgrade php-amqplib

### PR DESCRIPTION
**Pull request by masterjus**
Yesterday php-amqplib was upgraded to php8 with v3.0.0

### CODE REVIEW

File: Tests/AmqpConsumerTest.php [hunk 1]
```diff
@@ -68,15 +70,13 @@ public function testOnAcknowledgeShouldAcknowledgeMessage()
         $channel
             ->expects($this->once())
             ->method('basic_ack')
-            ->with(167)
-        ;
+            ->with(167);
```
Comment (masterjus) [inline Tests/AmqpConsumerTest.php:73]: autoformatter did this, I could change it back if necessary

**Comment by liasica** [comment]
thanks a lot.

**Comment by makasim** [comment]
Thanks for your pull request! We love contributions.

However, this repository is what we call a "subtree split": a read-only copy of one directory of the main enqueue-dev repository. It is used by Composer to allow developers to depend on specific Enqueue package.

If you want to contribute, you should instead open a pull request on the main repository:

https://github.com/php-enqueue/enqueue-dev

Read the contribution guide

https://github.com/php-enqueue/enqueue-dev/blob/master/docs/contribution.md

Thank you for your contribution!

**Comment by masterjus** [comment]
It's done https://github.com/php-enqueue/enqueue-dev/pull/1146

**Pull request closed without merging**